src/ddmd/backend/mscoffobj.c

```diff
@@ -1639,6 +1633,13 @@ segidx_t MsCoffObj::seg_xdata_comdat(symbol *sfunc)
 segidx_t MsCoffObj::seg_debugS()
 {
     // Probably should generate this lazilly, too.
+    if (segidx_debugS == UNKNOWN)
```

Comment:
The above comment seems obsolete now (and no need to correct the `lazily` typo ;)).

Reply:
Oops, somehow forgot to remove. Done now.